product/scanning.py

```diff
@@ -417,6 +417,7 @@ def _upload_image(
 
     def _transport_triages(self, triages, product_id):
         for triage in triages:
+            info(f"adding triage '{triage}' to product with id {product_id}')")
```

Comment:
this will cause too much noise. we might log to a file, maybe. or print the triage-IDs, if required
+ consider switching to python's default `logging` module